osgar/drivers/__init__.py

```diff
@@ -31,3 +31,11 @@
     realsense="osgar.drivers.realsense:RealSense",
 )
 
+
+def load_tests(loader, standard_tests, pattern):
```

Comment:
is it really necessary in each folder? I guess that there are some tests also in osgar/tools or osgar/lib?

Reply:
Yeah, I missed `osgar.lib`. It is the so called load_tests protocol: https://docs.python.org/3/library/unittest.html?highlight=unittest#load-tests-protocol so the `load_tests` function needs to be in each `__init__.py`. I am thinking how to reuse the implementation through some https://docs.python.org/3/library/functools.html?highlight=functools#functools.partial magic.

Reply:
I've done the refactoring.